src/command-handlers/passwords.ts

```diff
@@ -4,51 +4,72 @@ import { authenticator } from 'otplib';
 import winston from 'winston';
 import { AuthentifiantTransactionContent, BackupEditTransaction, Secrets, VaultCredential } from '../types';
 import { decryptTransactions } from '../modules/crypto';
-import { askCredentialChoice } from '../utils';
+import { askCredentialChoice, filterMatches } from '../utils';
 import { connectAndPrepare } from '../modules/database';
 
-export const runPassword = async (filters: string[] | null, options: { output: string | null }) => {
+export const runPassword = async (filters: string[] | null, options: { output: 'json' | 'clipboard' | 'password' }) => {
+    const { output } = options;
     const { db, secrets } = await connectAndPrepare({});
 
-    if (options.output === 'json') {
-        console.log(
-            JSON.stringify(
-                await selectCredentials({
-                    filters,
-                    secrets,
-                    output: options.output,
-                    db,
-                }),
-                null,
-                4
-            )
-        );
-    } else {
-        await getPassword({
-            filters,
-            secrets,
-            output: options.output,
-            db,
-        });
+    const clipboard = new Clipboard();
+    const selectedCredential = await selectCredential({ filters, secrets, db });
+
+    switch (output) {
+        case 'clipboard':
+            clipboard.setText(selectedCredential.password);
+            console.log(
+                `🔓 Password for "${selectedCredential.title || selectedCredential.url || 'N\\C'}" copied to clipboard!`
+            );
+
+            if (selectedCredential.otpSecret) {
+                const token = authenticator.generate(selectedCredential.otpSecret);
+                const timeRemaining = authenticator.timeRemaining();
+                console.log(`🔢 OTP code: ${token} \u001B[3m(expires in ${timeRemaining} seconds)\u001B[0m`);
```

Comment:
Don't you have an util for that kind of sequence, like Italic here.

Reply:
not by default, want another dependency? ahah